config.yml

```diff
@@ -33,10 +51,6 @@ workshops:
     name: Stream Adventure
     description: for a more advanced lesson
     link: https://github.com/substack/stream-adventure
-  -
-    name: Browserify Adventure
```

Comment:
Leave this block here, it's just for referencing in the arrays up above whenever we do this lesson at an event.